style.css

```diff
@@ -0,0 +1,210 @@
+  align-items: center;
+  justify-content: center;
+}
+
+#hero::after {
+  width: 100%;
+  height: 100%;
+  content: '';
+  position: absolute;
+  z-index: 10;
+  top: 0;
+  top: 0;
+  background: rgba(0, 0, 0, 0.5)
+}
+
+#hero video {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 5;
+  object-fit: cover;
+}
+
+#hero .content {
+  position: relative;
+  z-index: 20;
+  text-align: center;
+  color: white;
+}
+
+#hero .content h1 {
+  font-size: 1.5em;
+  Font-Family: 'Raleway', Sans-Serif;
+  margin-bottom: 0.5rem;
+
+}
+
+#hero .content h2 {
+  font-weight: 400;
+  font-size: 1rem;
+  margin: 0;
+  Font-Family: 'Open Sans', Sans-Serif;
+
+}
+
+#hero .content p {
+  font-size: 1rem;
+}
+
+#signbutton:hover {
+  cursor: wait;
+}
+/* form */
+.form {
+  position: relative;
+  z-index: 18;
+  display: inline-block;
+  text-align: left;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 12px 20px;
+
+}
+
+input[type=text],
+input[type=email],
+input[type=text],
+input[type=password],
+select,
+textarea {
+  width: 270px;
+  padding: 12px 20px;
+  margin: 8px 0;
+  display: inline-block;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+  font-size: 10px;
+}
+
+input[type=submit] {
+  width: 150px;
+  background-color: #4CAF50;
+  color: white;
+  padding: 14px 20px;
+  margin: 8px 0;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+
+}
+
+input[type=submit]:hover {
+  background-color: black;
+}
+
+.form-group {
+  border-radius: 5px;
+  background-color: #f2f2f2;
+  padding: 20px;
+  height: auto;
+  width: auto;
+  text-align: center;
+  font-size: 12px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center
+}
+/* Media queries */
+@media (min-width:768px) {
+  #hero .content h1 {
+    font-size: 2.5rem;
+  }
+
+  #hero .content h2 {
+    font-size: 1.75rem;
+  }
+}
+
+@media (min-width:992px) {
+  #hero .content h1 {
+    font-size: 4rem;
+  }
+
+  #hero .content h2 {
+    font-size: 2rem;
+  }
+}
+
+@media (min-width:1200px) {
+  .content h1 {
+    font-size: 4rem;
+  }
+
+  .content h2 {
+    font-size: 2.5rem;
+  }
+/* Accordion*/
+  .accordion {
+    position: relative;
+    z-index: 17;
+    display: inline-block;
+    text-align: left;
+    margin-left: auto;
+    margin-right: auto;
+    padding: 40 px;
+  }
+
+  .content-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+}
+
+.accordion-element {
+  border: 2px solid #999;
+  border-radius: 5px;
+  margin: 40px;
+  text-align: center;
+}
+
+.accordion-header {
+  border: 1px solid #999;
+  border-radius: 5px;
+  margin: 0;
+  padding: 15px;
+  font-size: 14px;
+  background-color: #f2f2f2;
+}
+
+.accordion-text {
+  border-top: 2px solid #999;
+  margin: 0;
+  padding: 15px;
+}
+
+.accordion-text-container {
+  height: 0;
+  overflow: hidden;
+  transition: height 0.3s ease;
+}
+/*Accordion open*/
+.active {
+  height: auto;
+}
```

Comment:
Your transitions would work as expected with a fixed height in this declaration, for example
```css
.active {
   height: 100px
}
```